src/store/flags.ts

```diff
@@ -63,12 +66,17 @@ const flags = createSlice({
             sidebar: false,
           },
           answers: {},
+          provenanceGraph: null,
         };
       }
       state.trialRecord[payload.trialId].valid[payload.location] =
         payload.status;
       const prev = state.trialRecord[payload.trialId].answers;
 
+      if(payload.provenanceGraph) {
```

Comment:
I prefer these if checks to be explicit

```suggestion
      if(payload.provenanceGraph !== null) {
```

Reply:
done